lisa/runners/legacy_runner.py

```diff
@@ -280,15 +279,27 @@ def _is_matched_infomation(
             result_vmsize = ""
             information_vmsize = ""
 
+        # TODO - Hard to match when publisher offer sku 1,
```

Comment:
The message can be more clear.
When v2 command specify both "latest" and explict versions, they may be mismatched with below logic. Since it's considered as cornner case to use them together, so leave it here.